.github/workflows/windows-build.bat

```diff
@@ -0,0 +1,44 @@
+REM kudos to Michal Karm Babacek <karm@fedoraproject.org>
+REM this script is from (https://github.com/modcluster/ci.modcluster.io)
```

Comment:
Is this script 1:1 with version in `ci.modcluster.io`? In that case let's just check out that repo as well, rather than repeat / copy the file.